### PR TITLE
Np 831 doi as uri

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: src/main/java/no/unit/nva/doi/transformer/MainHandler.java [hunk 1]
```diff
@@ -146,11 +146,9 @@ private JsonNode requestHeaders(JsonNode root) {
      */
     public static ObjectMapper createObjectMapper() {
```
Comment (janbj) [inline src/main/java/no/unit/nva/doi/transformer/MainHandler.java:147]: This is ObjectMapper for external data?

**Review by janbj — DISMISSED** [review]

File: src/main/java/no/unit/nva/doi/transformer/MainHandler.java (hunk 1 above)
Comment (axthosarouris) [reply src/main/java/no/unit/nva/doi/transformer/MainHandler.java:147]: No. This handler does not use commons. We should create another issue for refactoring it.

File: build.gradle [hunk 2]
```diff
@@ -18,17 +18,23 @@ targetCompatibility = JavaVersion.VERSION_11 // bytecode target version
...
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.2'
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: '2.10.2'
     implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.10.2'
     implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.10'
     implementation group: 'org.zalando', name: 'problem', version: '0.23.0'
     implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: '0.23.0'
 
-    implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: '0.3.1'
+
+    implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: '7daaa72e70'
```
Comment (axthosarouris) [inline build.gradle]: wating for nva-commons to be merged

**Review by janbj — DISMISSED** [review]

File: src/main/java/no/unit/nva/doi/transformer/CrossRefConverter.java [hunk 3]
```diff
@@ -162,10 +167,10 @@ private PublicationInstance extractPublicationInstance(CrossRefDocument document
 
     private String extractJournalTitle(CrossRefDocument document) {
         return Optional.ofNullable(document.getContainerTitle())
-                       .stream()
-                       .flatMap(Collection::stream)
-                       .findFirst()
-                       .orElse(null);
+            .stream()
+            .flatMap(Collection::stream)
+            .findFirst()
```
Comment (brinxmat) [inline src/main/java/no/unit/nva/doi/transformer/CrossRefConverter.java:172]: Are we only interested in the first title of many? Or do we expect only one?

**Review by brinxmat — APPROVED** [review]

**Pull request merged**